website/docs/mutation.md

```diff
@@ -0,0 +1,277 @@
+---
+id: mutation
+title: Gatekeeper mutation
+---
+
+The mutation feature allows gatekeeper to not only validate created kubernetes resources, but also modify them based on defined mutation policies.
+The feature is still in an alpha stage, so the final form can still changes. 
```

Comment:
Can we add a big "status: alpha" (or pre-alpha) at the top too so it's clear

Reply:
also, nit: "changes" -> "change"